src/NodeApi/Interop/JSCallbackDescriptor.cs

```diff
@@ -25,7 +31,18 @@ namespace Microsoft.JavaScript.NodeApi.Interop;
 
     public JSCallbackDescriptor(JSCallback callback, object? data = null)
     {
-        Callback = callback;
+        JSValueScope currentScope = JSValueScope.Current ??
+            throw new InvalidOperationException("No current scope.");
```

Comment:
Should we rather add a new property to `JSValueContext` that would do the verification. I believe we do this check in multiple places.

Reply:
I changed the existing `JSValueContext.Current` to throw if null. It works well, since it really never should be null.